pkg/sdn/plugin/node.go

```diff
@@ -63,6 +78,10 @@ func NewNodePlugin(pluginName string, osClient *osclient.Client, kClient *kclien
 		}
 	}
 
+	if err := writeCNIConfig(masterKubeConfig, hostname, mtu); err != nil {
```

Comment:
we should really be writing this at install time, and (at most) complaining if it is wrong at runtime